package.json

```diff
@@ -95,11 +95,11 @@
   "types": "./tsdef/index.d.ts",
   "scripts": {
     "lint:js": "eslint ./lib ./test/*.js ./docker/**/*.js ./test/framework/*.js ./test/lib/*.js ./test/integration/** ./test/unit/** ",
-    "lint:ts": "eslint ./tsdef/*.ts ./lib/**/*.ts",
+    "lint:ts": "eslint ./tsdef/*.ts ./lib/**/*.ts ./lib/*.ts",
     "lint": "npm-run-all lint:js lint:ts",
     "printVersion": "node --version && npm --version",
     "test:unit": "npm-run-all printVersion build:es5 test:unit:js test:unit:ts",
-    "test:unit:js": "nyc --report-dir=./coverage/js --extension=.js --include=lib/**/* --reporter=html --reporter=lcov --reporter=text mocha ./test/unit/index.js",
+    "test:unit:js": "nyc --report-dir=./coverage/js --extension=.js --include=lib/**/* --reporter=html --reporter=lcov --reporter=text mocha -r ts-node/register ./test/unit/index.js",
```

Comment:
Great question! Yes, I've tried using `require` instead but since we look at the non-compiled `createLocalTracks` file to do our unit tests, we need `ts-node/register`

Reply:
createLocalTracks's unit tests are under `test:unit:ts` script correct? So we don't need `ts-node/register` in `test:unit:js`?

Reply:
@charliesantos they're not since `createLocalTrack`'s tests are still a JS file.

Reply:
Ah that's true. In that case, we should probably combine `test:unit:js` and `test:unit:ts` since these two are running the same commands. Also, I think next step would be to convert the unit test file to TS (after this PR is approved. It can be another jira). We should do that before converting another public API.

Reply:
I'll create another ticket to convert the unit test file and pull it into this sprint then!

Reply:
Now that you converted the unit tests for createlocaltracks, can we now remove ts-node/register here?